html/changelogs/KingOfThePing - Fox.yml

```
@@ -0,0 +1,6 @@
+author: KingOfThePing
+
+delete-after: True
+
+changes:
+  - rscadd: "Replaced the fox sprite, because it was probably older than some players."
```

```suggestion
  - rscadd: "Replaced the fox sprite."
```
preferably informative only.

---

No fun allowed